benchmark/tests/looping.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-empty */
```

Comment:
There are 10 (https://github.com/middyjs/middy/blob/main/benchmark/tests/looping.js#L50, https://github.com/middyjs/middy/blob/main/benchmark/tests/looping.js#L20, etc) lines has the `no-empty` issue, so I think we can disable `no-empty` for whole file.